### DISCUSSION TITLE
[SF][A11Y]: Fast navigation using F6 for mentoring program is not work as expected

### DISCUSSION THREAD

**Discussion by sfsf-xweb-sh** [UI5/webcomponents#5199] [Q&A] [answered]
Hi colleague, 
When testing fast navigation in mentoring program details page, we found that it is unable to go back to global header through F6 from main content. The focus is stuck on "Join as mentee" button in following screenshot. 
 <img width="1084" alt="Screen Shot 2022-05-11 at 14 44 03" src="https://user-images.githubusercontent.com/78520316/167863250-fb305a21-7177-4a0c-a024-d9cf1edb4924.png">
When F6 is pressed from button, the focus should go back to the logo and keep looping from logo to "Join as mentee" and back to logo with subsequent F6 keypress. 
Could you help look at this issue? Thanks a lot!
Josephine Su

**Comment by fifoosid** [comment, marked as answer]
Hi @sfsf-xweb-sh 

Here is a [working example ](https://codesandbox.io/s/ui5-webcomponents-forked-qbs92b?file=/index.html)of the F6 Navigation. Some notes about the example: 

- You should import the ```import "@ui5/webcomponents-base/dist/features/F6Navigation.js";``` module in your application
- On the page you should have at least two groups, so that the navigation can cycle through them. In this example the first such group is the shellbar(it comes as a group be default), and the second is the div with attribute ```data-sap-ui-fastnavgroup="true"```(line 40 in the html file).

**Reply by sfsf-xweb-sh** [reply]
That’s great! Thank you so much!
